### DISCUSSION TITLE
do not jump back to document list when deleting a document from edit mode

### DISCUSSION THREAD

**Discussion by kurti500** [jonaswinkler/paperless-ng#913] [Feature Requests]
currently when deleting a document from the edit mode, paperless jumps back to the document list after pressing the delete button.
It would be more handy (at least when working on the list of todo/inbox items) if paperless would display the next document of the selection/document list.
Suggested behavior is similar to the "save/next" button.

Not sure if a dedicated "delete/next" button or a change of the current behavior or customization would be the best approach....

In addition a  delete button on each document card in the document list would be helpful, too.

Thanks for considering those adjustments.

**Comment by jonaswinkler** [comment]
> currently when deleting a document from the edit mode, paperless jumps back to the document list after pressing the delete button.
> It would be more handy (at least when working on the list of todo/inbox items) if paperless would display the next document of the selection/document list.
> Suggested behavior is similar to the "save/next" button.

Sounds like a very good idea. I'll get it done.

> Not sure if a dedicated "delete/next" button or a change of the current behavior or customization would be the best approach....

Don't need another button for a feature that's rarely used anyway.

> In addition a delete button on each document card in the document list would be helpful, too.

See above. The document cards are rather cramped already, too.
